File: python/0.stack/stack.py
stack = []

stack.append(5)
stack.append(4)
stack.append(3)
stack.append(2)
stack.append(1)
stack.pop()
stack.append(6)
stack.append(7)
stack.pop()

print(stack[::-1]) # 최상단 원소부터 출력하기
print(stack)       # 최하단 원소부터 출력하기

File: cpp/0.stack/stack.cpp
#include <iostream>
#include <stack>

using namespace std;

stack<int> s;

int main(int argc, char* argv[])
{
    s.push(5);
    s.push(4);
    s.push(3);
    s.push(2);
    s.push(1);
    s.pop();
    s.push(6);
    s.push(7);
    s.pop();

    while(!s.empty()) {
        cout << s.top() << ' ' ;
        s.pop();
    }
}